src/utils/createPptx/pptxConfiguration/slideElements.ts

```diff
@@ -67,8 +67,8 @@ export const imageStyling: PptxGenJS.ImageProps = {
 
 export const citeAsStyling: PptxGenJS.TextPropsOptions = {
   x: '70%',
-  y: '85%',
-  fontSize: 12,
+  y: '90%',
```

Comment:
Please express this with variables instead, probably break out mainContent's `h` to a variable MAIN_CONTENT_HEIGHT, and have this y-value be like Y_PADDING + MAIN_CONTENT_HEIGHT + CITE_AS_Y_MARGIN.

Also give this its own h; CITE_AS_HEIGHT and have it be remainingHeight(Y_PADDING * 2 + MAIN_CONTENT_HEIGHT + 2 * CITE_AS_Y_MARGIN)

(Take all values above with a grain of salt, it probably needs some fine-tuning)

Reply:
Done